code/datums/uplink_item.dm

```diff
@@ -415,7 +415,7 @@ var/list/uplink_items = list()
 	desc = "A small, easily concealable device. It can be applied to an open airlock panel, and the next person to open that airlock will be knocked down in an explosion. The airlock's maintenance panel will also be destroyed by this."
 	item = /obj/item/device/doorCharge
 	cost = 5
-	surplus = 6
+	surplus = 10

```

Comment:
This significant increase isn't in your list of changes, did you mean to make it 10?